gdraw/Makefile.am

```diff
@@ -41,7 +41,8 @@ libgdraw_la_SOURCES = choosericons.c ctlvalues.c drawboxborder.c	\
 	gdrawable.c gspacer.c xkeysyms_unicode.c colorP.h gdrawP.h	\
 	gimagebmpP.h gresourceP.h gxcdrawP.h fontP.h ggadgetP.h		\
 	gpsdrawP.h gwidgetP.h gxdrawP.h hotkeys.c hotkeys.h		\
-	div_tables.c ggdkdraw.c ggdkcdraw.c ggdkdrawlogger.c ggdkdrawP.h
+	div_tables.c ggdkdraw.c ggdkcdraw.c ggdkdrawlogger.c            \
+	gtkbbr.c gtkbridge.c ggdkdrawP.h
```

Comment:
`gtkbbr.c` is a (probably) temporary code context for moving between FontForge and GTK. It's basically an include file issue; if you try to include all of everything it gets ugly. Easier to have a neutral location to sort through pointer chains.